src/skin/popup.css

```diff
@@ -148,34 +148,46 @@ a.removeOrigin:hover {
     display: inline-block;
 }
 
-#privacyBadgerHeader {
+#privacy-badger-header {
+    display: flex;
     color: #505050;
     font-size: 16px;
     margin-bottom: 5px;
 }
 
-#title {
+#badger-logo-div {
+    width: 48px;
+}
+
+#badger-logo-div, #badger-title-div {
     padding-top: 4px;
 }
 
-#header-image-stack {
-    display: table;
+#badger-title-div {
+    flex: 1;
     margin: 2px 10px 0 10px;
-    line-height: 1.0;
+    line-height: 0.8;
```

Comment:
The gap between the EFF logo image and the Privacy Badger title text is now noticeably smaller on some browsers/platforms.